lib/explorer/polars_backend/series.ex

```diff
@@ -422,6 +422,39 @@ defmodule Explorer.PolarsBackend.Series do
   @impl true
   def unary_not(%Series{} = series), do: Shared.apply_series(series, :s_not, [])
 
+  # Strings
+
+  @impl true
+  def contains(series, %Regex{source: pattern}),
+    do: Shared.apply_series(series, :s_contains, [pattern])
```

Comment:
You also need to pass the opts field, so you get the modifiers, such as:

    ~r/foo/i

For case insensitive. You may need to check if all Elixir modifiers map to Rust modifiers.

Reply:
Another concern is that regexes in Elixir are not Unicode by default but they seem to be in Polars. I am not even sure if they have a non-Unicode variant. So we should perhaps expect "u" to be set in `opts`? Such as `~r/foo/u` or somehow [disable Unicode](https://docs.rs/regex/latest/regex/#opt-out-of-unicode-support) if "u" is not specified.

Reply:
Perhaps remove the Regex support from this PR we work on the foundation for it in the next one?

Reply:
Sure, I'll take it out for now.